cli/integration_tests/basic_monorepo/dry_run.t

```diff
@@ -19,37 +19,46 @@ Setup
-    Command                 = echo 'building'                                                                                        
-    Outputs                 =                                                                                                        
-    Log File                = packages/util/.turbo/turbo-build.log                                                                   
-    Dependencies            =                                                                                                        
-    Dependendents           =                                                                                                        
-    Inputs Files Considered = 1                                                                                                      
-    ResolvedTaskDefinition  = {"outputs":[],"cache":true,"dependsOn":[],"inputs":[],"outputMode":"full","env":[],"persistent":false} 
+    Task                             = build                                                                                                            
+    Package                          = util                                                                                                             
+    Hash                             = 1a3651e1149bfaf7                                                                                                 
+    Cached (Local)                   = false                                                                                                            
+    Cached (Remote)                  = false                                                                                                            
+    Directory                        = packages/util                                                                                                    
+    Command                          = echo 'building'                                                                                                  
+    Outputs                          =                                                                                                                  
+    Log File                         = packages/util/.turbo/turbo-build.log                                                                             
+    Dependencies                     =                                                                                                                  
+    Dependendents                    =                                                                                                                  
+    Inputs Files Considered          = 1                                                                                                                
+    Configured Environment Variables = NODE_ENV=e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855                                        
```

Comment:
One thing that's a little weird, is that if the env var is configured, even if it is not _set_, it adds an empty string into the taskHashInput. Since it is part of the task hash calculation, I'm opting to have it show up here also. But this may be confusing to see.